public/css/custom-asciinema-player.css

```diff
@@ -1,5 +1,5 @@
 .asciinema-terminal {
-  height: 320px !important;
+  height: 340px !important;
```

Comment:
Is there any thing we can do about the !important declarations being used in this entire file ?

Reply:
Asciinema uses inline CSS. In order to override that we need to use `!important`

Reply:
okay.